scripts/meson-kernel-aarch64-managarm.cross-file

```diff
@@ -0,0 +1,57 @@
+[binaries]
+c = 'clang'
+cpp = 'clang++'
+cpp_ld = 'lld'
+c_ld = 'lld'
+as = 'aarch64-none-elf-as'
+strip = 'aarch64-none-elf-strip'
+objcopy = 'aarch64-none-elf-objcopy'
+size = 'aarch64-none-elf-size'
+pkgconfig = 'pkgconfig'
+
+[properties]
+needs_exe_wrapper = true
+
+sizeof_int = 4
+sizeof_wchar_t = 4
+sizeof_void* = 8
+
+alignment_char = 1
+alignment_void* = 8
+alignment_double = 8
+
+has_function_printf = false
+
+cpp_args = [
+	'-target', 'aarch64-managarm-kernel',
+	'-ffreestanding',
+	'-mcpu=cortex-a72',
+	'-fno-stack-protector',
+	'-static', '-fno-plt', '-fno-pic', '-fno-PIE',
+	'-gcc-toolchain', '_BUILD_ROOT_/tools/kernel-gcc',
+	'-nostdlib']
+
+c_args = [
+	'-target', 'aarch64-managarm-kernel',
+	'-ffreestanding',
+	'-mcpu=cortex-a72',
+	'-fno-stack-protector',
+	'-static', '-fno-plt', '-fno-pic', '-fno-PIE',
+	'-gcc-toolchain', '_BUILD_ROOT_/tools/kernel-gcc',
+	'-nostdlib']
```

Comment:
Doesn't the meson.build already pass `-ffreestanding`, `-fno-stack-protector`, ...? On x86, we need much fewer flags here.